test-integ/peering_commontopo/ac5_2_test.go

```diff
@@ -12,79 +12,54 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-// This tests prepared query failover across 3 clusters. The following test case covers:
-// 1. Create a prepared query with failover target to DC2 and DC3 cluster.
-// Test will fail if not run in parallel
 type preparedQueryFailoverSuite struct {
-	DC   string
-	Peer string
-
-	nodeServer   []*topology.Node
-	dc3ServerSID topology.ServiceID
-
-	serverSID    topology.ServiceID
-	clientSID    topology.ServiceID
-	dc3APIClient *api.Client
+	clientSID  topology.ServiceID
+	serverSID  topology.ServiceID
+	nodeServer topology.NodeID
+	ct         *commonTopo
 }
 
-var (
-	preparedQueryFailoverSuites []*preparedQueryFailoverSuite = []*preparedQueryFailoverSuite{
-		{DC: "dc1", Peer: "dc2"},
-		{DC: "dc2", Peer: "dc1"},
-	}
-	query *api.PreparedQuery
-	def   *api.PreparedQueryDefinition
-)
+var ac5Context = make(map[nodeKey]preparedQueryFailoverSuite)
 
 func TestPreparedQueryFailoverSuite(t *testing.T) {
-	var querySuite *preparedQueryFailoverSuite
+	if !*FlagNoReuseCommonTopo {
+		t.Skip("NoReuseCommonTopo unset")
+	}
```

Comment:
```suggestion
```
Too late, but since this test isn't in `commonTopoSuites` (as it shouldn't be, since it's destructive), we don't want it to require the flag